newsLetter/newsletter.py

```diff
@@ -255,6 +257,101 @@ def format_newsletter(content: str)->tuple[str, str]:
     newsletter_content += "Stay informed with Wall-E's tech updates, and see you back here tomorrow!\n\n---\n\n[🔊 Listen to the Full Podcast Episode Here](https://aibriefingroom.podbean.com/)"
     return newsletter_content
 
+def email_exists(service, email: str) -> bool:
+    SPREADSHEET_ID = os.getenv(SPREADSHEET_SRC)
```

Comment:
Maybe consider renaming with SPREADSHEET_ID and NEW_SPREADSHEET_ID? Add an experiment flag here to control it maybe?

Reply:
Maybe consider putting the flag into a `config.json`

Reply:
Okayyy